README.md

```diff
@@ -16,6 +16,23 @@ for more background, objectives, key use cases.
 
 ## Development
 
+### Docker
+
+You may run the Breach Alerts service with [Docker](https://www.docker.com/).
+
+If you have Docker installed, you can:
+
+1. Build the container image:
+
+    `docker build -t blurts-server .`
+    
+2. Run the container:
+
+    `docker run -d -p 6060:6060 blurts-server`
```

Comment:
Never sure the best way to do this... Like, should we create some handy aliases for these in package.json? A la,

```js
"docker:build": "docker build -t blurts-server .",
"docker:run": "docker run -d -p 6060:6060 blurts-server",
```

Then we'd just need to run <kbd>$ npm run docker:build</kbd> or <kbd>$ npm run docker:run</kbd>.